frontend/www/js/omegaup/components/course/ViewStudent.vue

```diff
@@ -242,6 +249,13 @@ export default class CourseViewStudent extends Vue {
       return;
     }
     this.selectedProblem = found;
+    this.selectedRun = found.runs.length ? found.runs[0] : null;
```

Comment:
```suggestion
    this.selectedRun = found.runs.?[0] ?? null;
```

el operator de [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#array_item_access_with_optional_chaining) funciona para arreglos!